### PR TITLE
Switch to variables for namespace selector height and

### PR DESCRIPTION
**Pull request by sg00dwin**
... use flexbox align-items to vertically center the import yaml link.

screen shot 2018-10-10 at 3 51 56 pm

### CODE REVIEW

**Comment by spadgett** [comment]
/assign @rhamilto 

FYI, we will probably switch this to a dropdown with more options.

**Comment by rhamilto** [comment]
I don't think the height is necessary.  I'd opened #638 to remove it.

File: frontend/public/components/_dropdown.scss [hunk 1]
```diff
@@ -264,8 +265,11 @@ $color-bookmarker: #DDD;
 }
 
 .co-ns-selector {
-  height: 42px;
+  height: $co-namespace-selector-mobile;
   transition: 0.4s;
```
Comment (rhamilto) [inline frontend/public/components/_dropdown.scss]: Remove the transition to avoid the unwanted animation

**Comment by rhamilto** [comment]
Since I'm replacing the link with a dropdown, I propose I fold these changes in to mine?

**Comment by spadgett** [comment]
I believe this was folded into Robb's changes. If I'm wrong, feel free to reopen.

/close

**Pull request closed without merging**